src/axom/slic/CMakeLists.txt

```diff
@@ -75,6 +75,13 @@ blt_add_library(NAME        slic
                 OBJECT      TRUE
                 )
 
+# If we're compiling as a dynamic-link library, define AXOM_EXPORTS so we export symbols.
+if(BUILD_SHARED_LIBS)
```

Comment:
Should this be `if (WIN32 AND BUILD_SHARED_LIBS)` ?

Reply:
hmm, ideally -- they should be defined in all cases when compiling, regardless of static vs shared vs platform.